src/components/GoalForm/GoalForm.tsx

```diff
@@ -60,9 +59,15 @@ interface GoalFormProps extends React.HTMLAttributes<HTMLDivElement> {
     id?: string;
     tip?: React.ReactNode;
 
-    onSumbit: (fields: z.infer<GoalFormProps['validitySchema']>) => void;
+    onSubmit: (fields: z.infer<GoalFormProps['validitySchema']>) => void;
 }
 
+const defaultPriority = {
```

Comment:
No way! Let's do that via req. Check flow dropdown for ex. Also add comment with issue which @9teen90nine mentioned.